proto/regen/ecocredit/v1/events.proto

```diff
@@ -193,4 +194,19 @@ message EventBridge {
 
   // contract is the contract address.
   string contract = 3;
-}
\ No newline at end of file
+
+  // amount is the amount of credits.
+  string amount = 4;
+}
+
+// EventBridgeReceive is emitted when credits are bridged from another chain.
+message EventBridgeReceive {
```

Comment:
What's the use case for this event? I'm wondering if we should also add `amount` here too?

Reply:
`amount` would be redundant with `EventMint`, which will be emitted from `Msg/CreateBatch` or `Msg/MintBatchCredits`. 

In general, we are taking a lean approach to event fields based on how events are written in the sdk and `EventBridge` is a special case because we want to make all necessary information easily accessible to the bridge service.

We would ideally include an event specific to each message so that we can track the execution of each message and the two fields here (project id and batch denom) would be the most relevant information to `Msg/BridgeReceive`.

See #1035 and #1086 for reference.

Reply:
If anything, we could probably remove project id but it still seemed relevant.

Reply:
If anything, we could probably remove project id but it still seems relevant.